BuildaGitServer/Base/Authentication.swift

```diff
@@ -69,4 +73,16 @@ extension ProjectAuthenticator: KeychainStringSerializable {
             self.secret
             ].joinWithSeparator(":")
     }
+
+    internal static func createEnterpriseService(host: String) -> GitService? {
+        guard let url = NSURL.init(string: "http://\(host)") else { return nil }
+        do {
+            let response = try NSString.init(contentsOfURL: url, encoding: NSASCIIStringEncoding)
+            if response.lowercaseString.containsString("github") {
```

Comment:
I suspect it's customized per Enterprise GitHub installation.  For our company it's a home page that looks much like I remember www.github.com looking last year.  The html happens to include the word "github" in several places.  It's an extremely fragile way of testing for an Enterprise GitHub server, but it was the best I could come up with.